From 3e2a0b037d9ba1697b4bc3a3cfe71ed9101cd898 Mon Sep 17 00:00:00 2001
From: Nava kishore Manne <nava.manne@xilinx.com>
Date: Mon, 3 Feb 2020 15:46:08 +0530
Subject: [PATCH 1116/1851] arm64: zynqmp: Add support for ZynqMP RSA H/W
 accelerator

commit 11205cb74e2ff24c7900e9062551654e67dd7e86 from
https://github.com/Xilinx/linux-xlnx.git

This patch adds support for ZynqMP RSA H/W accelerator.

Signed-off-by: Nava kishore Manne <nava.manne@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: not-upstreamable
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 arch/arm64/boot/dts/xilinx/zynqmp.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/xilinx/zynqmp.dtsi b/arch/arm64/boot/dts/xilinx/zynqmp.dtsi
index 52fb6e02ac99..af120646e815 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp.dtsi
+++ b/arch/arm64/boot/dts/xilinx/zynqmp.dtsi
@@ -166,6 +166,10 @@
 			xlnx_keccak_384: sha384 {
 				compatible = "xlnx,zynqmp-keccak-384";
 			};
+
+			xlnx_rsa: zynqmp_rsa {
+				compatible = "xlnx,zynqmp-rsa";
+			};
 		};
 	};
 
-- 
2.31.1

